resources/views/nova/help/travel/assignment/member.blade.php

```diff
@@ -0,0 +1 @@
+Only members that have recently paid dues are shown here by default. If you need to create an assignment for a member that hasn't paid dues, go to their user details page and click <strong>Create Trip Assignment</strong> there.
```

Comment:
maybe remove "by default" for brevity?

also is "recently paid dues" simplifying some complicated logic? is it anyone considered active (vs. access active but not active? I just feel like the wording could be more precise

Reply:
It is just access active, yes, but I don't think that's helpful information for non-technical users.